src/styles/_catalog.scss

```diff
@@ -1,22 +1,41 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
 
-  max-width: 944px;
+  width: 944px;
```

Comment:
It should be max-width. Because your catalog has incorrect behavior 
<img width="634" alt="Знімок екрана 2023-01-03 о 14 33 17" src="https://user-images.githubusercontent.com/117587463/210358095-a5699a0f-efbb-4e9c-88a4-77be449f6165.png">

```suggestion
  max-width: 944px;
```